README.md

```diff
@@ -1,5 +1,20 @@
 # Regulatory R Package Repository
 
+Using R in submissions to healthcare regulators is challenging since the 
+quality of any packages used must be ensured and evidence of ensuring this
+quality must be made available on request. 
+This working group strives to identify and prototype at least one technical
+framework that can support a transparent, open, dynamic, cross-industry approach
+of establishing and maintaining a 'repository' of core R packages with
+accompanying evidence of their quality that can be used to simplify necessary
+in-house validation processes as much as possible.
+Such a cross-industry 'repository' could help with burden-sharing of
+validation efforts, improve quality via transparent, open peer review, and
+de-risk the use of important R packages for regulatory submissions.
```

Comment:
Generalizing the "important" thing just to reflect that this could pertain to all public packages.

"important" can be a bit ambiguous - is it important in how many reverse dependencies it supports? important for regulatory reporting? important due to frequency of use? I think it's fine to keep it open-ended for now.

```suggestion
de-risk the use of public R packages for regulatory submissions.
```